packages/calcite-components/src/components/table/table.e2e.ts

```diff
@@ -1642,9 +1689,8 @@ describe("keyboard navigation", () => {
     await page.keyboard.press("PageUp");
     await page.waitForChanges();
     expect(await getFocusedElementProp(page, "id")).toBe("head-1b");
-    await page.keyboard.down("ControlRight");
+    page.keyboard.press("ControlRight");
```

Comment:
This should use `await` and changing it to `press` will cause it to no longer go into the expected `End` + `ControlRight` path [here](https://github.com/Esri/calcite-design-system/blob/b2c976214774490b5a313e1fafc67415fe212b37/packages/calcite-components/src/components/table-row/table-row.tsx#L248-L251) (i.e., `ControlRight` is pressed and released before `End` is pressed).